kotlin.kt

```diff
@@ -1,20 +1,16 @@
-public fun main(args: Array<String>) {
-	for (i in 1..100) {
-		fizzBuzz(i)
-	}
+inline fun Int.divisibleBy(x: Int) = this % x == 0
+
+fun main(args: Array<String>) {
+	(1..100).forEach(::fizzBuzz)
 }
 
 fun fizzBuzz(n: Int) {
-	val divisibleBy3 = n % 3 == 0
-	val divisibleBy5 = n % 5 == 0
-
-	if (divisibleBy3 && divisibleBy5) {
-		println("FizzBuzz")
-	} else if (divisibleBy5) {
-		println("Buzz")
-	} else if (divisibleBy3) {
-		println("Fizz")
-	} else {
-		println(n)
-	}
+	println(
+		when (true) {
```

Comment:
Sorry, I forgot you can comment inline on the file.
What's this doing? It seems fairly clear to me that it's not a `while(true)` which I think would be bad.